googleCalender.js

```diff
@@ -0,0 +1,427 @@
+const max = 50;
+async function usersFullSync({ dataStore, client }) {
+  const responseData = await client.fetch(
+    "admin/directory/v1/users?customer=my_customer&projection=full&viewType=admin_view"
+  );
+
+  if (!responseData.ok) {
+    throw new Error(
+      `Events sync failed ${responseData.status}:${responseData.statusText}.`
+    );
+  }
+  const data = await responseData.json();
+
+  for (let user of data.users) {
```

Comment:
I believe the user should be constant. Since we are not assigning any values after declaring.
```suggestion
  for (const user of data.users) {
```